### DISCUSSION TITLE
Instancio is interested in joining

### DISCUSSION THREAD

**Discussion by armandino** [commonhaus/foundation#138] [Joining Commonhaus]
### Project information

- **Project name**: Instancio
- **Project website**: https://www.instancio.org
- **Code repository**: https://github.com/instancio/instancio/
- **License**: Apache 2.0

### Describe your project

Instancio is a Java library for automating data setup in tests. It's main is to help developers write tests faster, and to reduce the amount of boilerplate code when setting up test data. It does so by populating objects with random data, which can be fully reproduced in case of test failure.

The library is open source and is not backed by any company. It is distributed under the Apache v2 license.

**Comment by ebullient** [comment]
Please use this discussion (not necessarily this sub-thread) to ask any other questions you may have.

- [ ] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting). You may have seen some PRs go through as I work with the current EGC to create a record of approval with the newly established EGC) to ratify them and you're welcome to weigh in on these.
- [ ] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [x] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)

**Reply by armandino** [reply]
@ebullient thank you, submitted a PR.

- #157

**Comment by ebullient** [comment]
Resolved (for now) by #157 

Let's revisit this conversation again around April of next year. 

🙏 🫂
